cPP_DSC.adoc

```diff
@@ -1856,7 +1855,7 @@ _Application Note {counter:remark_count}_:: _If FPT_MFW_EXT.1 selects [.underlin
 +
 _Recall that resetting a TOE to factory state also wipes all user data, but may not wipe out pre-installed SDOs. Configuring authorization policies includes setting policies for allowed access to SDOs._
 +
-_Protections for pre-installed SDEs/SDOs come through the firmware as well as through firmware updates. In the same vein, the authorized updates may also affect the SDEs as well, if the vendor so chooses. One could say that the authorized update binds the attributes present in the functionality of the firmware to the pre-installed SDEs._
+_Protections for pre-installed SDEs/SDOs come through the firmware as well as through firmware updates. In the same vein, the authorized updates may also affect the SDOs as well, if the vendor so chooses. One could say that the authorized update binds the attributes present in the functionality of the firmware to the pre-installed SDOs._
```

Comment:
I believe this should be "SDOs" because the rest of the note is talking about SDOs. I might be wrong though.

Reply:
Was resolved in some previous PR.